### DISCUSSION TITLE
Add Keep alive on tab component vue

### DISCUSSION THREAD

**Discussion by hi-reeve** [tailwindlabs/headlessui#1048] [Ideas]
is it possible to make the tab component to be keep-alive using the built-in keep-alive component from vue?

**Comment by mitar** [comment]
See #743.

**Reply by mitar** [reply]
I made also #2111.

**Comment by mitar** [comment]
I agree, the `unmounted=false` is similar to using built-in keep-alive component, but not exactly the same. `unmounted=false` uses `display: hidden` to keep DOM around, while keep-alive component removes DOM from the document, while keeping DOM objects around to be re-inserted later. I think it would be great if keep-alive built-in component would work with tab panels, too.
